pkg/index/job/correction/service/corrector_test.go

```diff
@@ -472,8 +466,7 @@ func Test_correct_correctReplica(t *testing.T) {
 //
 // func TestNew(t *testing.T) {
 // 	type args struct {
-// 		cfg        *config.Data
-// 		discoverer discoverer.Client
+// 		opts []Option
```

Comment:
> **NOTE**
I regenerated the test code because the field variables in the structure (`corrector`) have changed 🙏